client-scripts-api.md

```diff
@@ -277,3 +277,111 @@ Payload:
   value: true|false
 }
 ```
+
+### `routeChange`
+
+This event is triggered when a change in the internal router happens.
+
+Payload:
+
+```js
+{
+  location: {
+    pathname: '/products',
+    search: '?page=0',
```

Comment:
❓ not sure i get the name search here, wouldnt query be sufficient?

Reply:
Query is not always there. Depends if it's transformed or not.

Reply:
ok i dont understand what you mean 😅 
if query is the query param, what is search? it looks like its just the query params in string form

Reply:
yes, but the library not always transforms the search string to `query` It's not transformed by us but by the router lib.
I just figured it's not always there, so I included the raw one as well